components/Templates/includes/functions-view_template.php

```diff
@@ -522,9 +577,13 @@ function frontier_prefilter_template( $code, $template, $pod ) {
 							$field = trim( $matches[2][ $key ] );
 						}
 						if ( false !== strpos( $field, '.' ) ) {
-							$path  = explode( '.', $field );
-							$field = array_pop( $path );
-							$ID    = '{@' . implode( '.', $path ) . '.' . $pod->api->pod_data['field_id'] . '}';
+							if ( 'if' === $command ) {
+								$ID = '{@' . $pod->api->pod_data['field_id'] . '}';
```

Comment:
:no_entry_sign: Variable "$ID" is not in valid snake_case format, try "$i_d"

<sub>hash: 8d046c4ee0dd7d136046a6115c7bcf4e</sub>